Kwartracker/Scenes/Reusable/MenuView.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  MenuView.swift
+//  Kwartracker
+//
+//  Created by Veverly Mae Ricaza on 5/24/21.
+//
+
+import SwiftUI
+
+struct MenuView: View {
```

Comment:
It seems like this view is not being use?

Reply:
Deleted already